### PR TITLE
Adding MS phishing sim domains

### PR DESCRIPTION
**Pull request by aidenmitchell**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -108,6 +108,15 @@
       "includes_header": true,
       "format": "hostname"
     },
+    {
+      "file": "ms_phishing_simulation_domains.txt",
+      "identifier": "ms_phishing_simulation_domains",
```
Comment (rw-access) [inline manifest.json:113]: seems a little too specialized for lists? I don't think we have other ones in here

Comment (jkamdjou) [reply manifest.json:113]: agree, there's no use for this i can think of outside of the global exclusion. in which case, we should prob hard code it in the Exclusion rule

Comment (jkamdjou) [reply manifest.json:113]: @aidenmitchell can you turn this into a standalone rule that we can add as a Global Exclusion instead?

Comment (jkamdjou) [reply manifest.json:113]: @aidenmitchell can you turn this into a standalone rule that we can add as a Global Exclusion instead?

(fysa: @morriscode )

**Pull request closed without merging**